actioncable/lib/action_cable/engine.rb

```diff
@@ -31,6 +32,10 @@ class Railtie < Rails::Engine # :nodoc:
           self.cable = Rails.application.config_for(config_path).with_indifferent_access
         end
 
+        self.connection_class = ApplicationCable::Connection
+
+        channel_load_paths << Rails.root.join('app/channels')
```

Comment:
Ah, cool! You mean the root to join onto? I.e. `Rails.application.paths.path.join('app/channels')`?.